scripts/missions/toau/08_A_Mercenary_Life.lua

```diff
@@ -32,8 +32,10 @@ mission.sections =
 
     {
         check = function(player, currentMission, missionStatus, vars)
-            return currentMission == mission.missionId and
-                not mission:getMustZone(player)
+            return
+                currentMission == mission.missionId
+                and os.time() >= player:getCharVar('WesterlyWinds_date')
```

Comment:
And then down here we can go
```suggestion
                and os.time() >= mission:getVar(player, 'Stage')
```

Reply:
does ```mission:getVar(player, 'Stage')``` access a player variable set on the previous mission?